README.md

```diff
@@ -1 +1,156 @@
-# CATALOG
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents 
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** is a console application that allows users to manage collections of things they own. It contains book, music album, movie and game classes. Everything is based on the UML diagram.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<!-- Features -->
```

Comment:
- [ ] However, I am not able to see any link for a video presentation so I would kindly ask you to add a video presentation of the project to your README file. given us the walkthrough if the project and demonstrated what you have have and what you are most proud of implementing in the project.


 Project requirement 👇

![image](https://github.com/2004-okari/CATALOG/assets/58553711/380dd2cd-f781-435f-a6b9-71b99e33bc76)